src/templates/objects/list.html

```diff
@@ -269,52 +284,16 @@
                 class="w-full pl-12 p-3 text-base font-semibold text-gray-600 focus:outline-none border border-gray-300 rounded-md" />
         </div>
     </div>
-    <div class="flex flex-nowrap gap-x-5">
-        <!-- <div class="flex bg-[#181c320d] py-3.5 px-4 md:px-6 rounded-md">
+    <!-- <div class="flex flex-nowrap gap-x-5">
```

Comment:
Let's remove this comment if not needed

Reply:
This feature needs to be implemented in the future.